static/js/TopicLandingPage/TopicLandingSearch.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {GeneralAutocomplete} from "../GeneralAutocomplete";
+import Sefaria from "../sefaria/sefaria";
+import {SearchButton} from "../Misc";
+
+const getSuggestions = async (input) => {
+    if (input.length <=1) {
+      return [];
+    }
+    const word = input.trim();
+
+    const _getFormattedPath = (slug, lang) => {
+      const categories = Sefaria.topicTocCategories(slug);
+      if(!categories){return ""}
+      const titles = categories.map((cat) => cat[lang]);
+      return "("+ titles.join(` > `) + ")";
+    }
+
+    const parseSuggestions = (d, lang) => {
+      let topics = [];
+      if (d[1].length > 0) {
+        topics = d[1].slice(0, 10).map((e) => ({
+          title: "# " + e.title + " " + _getFormattedPath(e.key, lang),
+          key: e.key,
+        }));
+      }
+      const dummySuggestion = {title: "# Dummy", key: 'dummy'};
+      topics = topics.concat([dummySuggestion, dummySuggestion, dummySuggestion, dummySuggestion, dummySuggestion])
+      return topics;
+    };
+
+    const isInputHebrew = Sefaria.hebrew.isHebrew(word);
+    const lang = isInputHebrew? 'he' : 'en';
+
+    const rawCompletions = await Sefaria.getTopicCompletions(word);
+    const completionObjects = parseSuggestions(rawCompletions, lang)
+    return completionObjects.map((suggestion) => ({
+      text: suggestion.title,
+      slug: suggestion.key,
+    }));
+  };
+
+
+const renderItem = (openTopic, item, index, highlightedIndex, getItemProps)=>{
+  const isHighlighted = index === highlightedIndex;
+  const highlightedClassString = isHighlighted ? "highlighted" : '';
```

Comment:
use package `classNames`. e.g. like this `  let elemclasses = classNames({"int-en": !isHebrew, "int-he": isHebrew});`